movies/src/resources/doc.jsonld

```diff
@@ -1,15 +1,50 @@
 {
   "@context": "/context.jsonld",
   "@id": "/doc",
-  "@type": "hydra:ApiDocumentation",
-  "hydra:title": "Hydra Movies API",
-  "hydra:description": "This API shows how a basic CRUD (Create, Read, Update, Delete) API can be designed using Hydra Core Vocabulary.",
-  "hydra:entrypoint": "/",
-  "hydra:supportedClass": [
+  "@type": "ApiDocumentation",
+  "title": "Hydra Movies API",
+  "description": "This API shows how a basic CRUD (Create, Read, Update, Delete) API can be designed using Hydra Core Vocabulary.",
+  "entrypoint": "/",
+  "supportedClass": [
+    {
+      "@id": "schema:EntryPoint",
+      "@type": "Class",
+      "title": "API entrypoint",
+      "description": "Every capability of the API can be reached from here by following links."
+    },
+    {
+      "@id": "mov:Movies",
+      "@type": "Class",
+      "title": "Movies",
+      "description": "Allows to access and manage movies that are available via the API.",
+      "supportedOperation": {
+        "@id": "mov:create-movie",
+        "@type": [ "Operation", "schema:CreateAction" ],
+        "title": "Create movie",
+        "description": "Creates a new movie",
+        "method": "POST",
+        "expects": "schema:Movie",
+        "returns": "schema:Movie",
+        "possibleStatus": [
```

Comment:
While switching to the provided context, I noticed I was not spec-conformant here. I copied some example which seems to be outdated. I am still not sure if describing the status code this way is as it is meant to be. Any comments?

Reply:
I'm not sure anymore. We should analyse the history behind these. The spec still mentions both properties. Let's not deal statuses them in this PR

Reply:
Oh, I see what you did there. Yes, I think it's good now

Reply:
I think we should open a spec-issue to clarify the usage of those terms and types in prose. Do you agree?

Reply:
Agreed, let's settle it once and for all. We should revisit the practical application of status code description for clients actually.

Reply:
https://github.com/HydraCG/Specifications/issues/191